core/src/main/java/org/elasticsearch/common/document/DocumentField.java

```diff
@@ -36,34 +36,46 @@
 import static org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
 import static org.elasticsearch.common.xcontent.XContentParserUtils.parseStoredFieldsValue;
 
-public class GetField implements Streamable, ToXContent, Iterable<Object> {
+public class DocumentField implements Streamable, ToXContent, Iterable<Object> {
```

Comment:
nit: Can we have some javadoc here, maybe a bit longer that the current SearchHitField javadoc that also states that this is used in both SearchHit and GetResult now?